### PR TITLE
System stats reported by SystemManager (exposed via SOAP).

### PR DESCRIPTION
**Pull request by menivaitsi**
This service relies upon psutil for cpu, disk, memory and swap stats.

### CODE REVIEW

**Comment by menivaitsi** [comment]
CI Build: https://ocd.appscale.com:8080/job/Daily%20Build/1173/

File: InfrastructureManager/system_manager.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    return json.dumps(mem_stats_dict)
+
+  def get_monit_summary(self):
+    """ Retrieves Monit's summary on this node.
+
+    Returns:
+      A dictionary containing Monit's summary as a string.
+    """
+    monit_stats = subprocess.check_output(["monit", "summary"])
+    monit_stats_dict = { JSONTags.MONIT : monit_stats}
```
Comment (obino) [inline InfrastructureManager/system_manager.py]: Should this be a bit more processed? ie the list of services we have running on this node? or the list of failed services we have on this node?

File: InfrastructureManager/infrastructure_manager_service.py [hunk 2]
```diff
@@ -99,10 +109,10 @@ def start(self):
     threading requirements
     """
     if self.started:
-      utils.log('Warning - Start called on already running server')
+      logging.warn('Start called on already running server')
     else:
-      utils.log('Starting AppScale Infrastructure Manager on port: ' +
-                str(self.port))
+      logging.info('Starting AppScale Infrastructure Manager on port: '
+        '{}'.format(str(self.port)))
```
Comment (cdonati) [inline InfrastructureManager/infrastructure_manager_service.py]: `format` does an implicit string conversion.

File: debian/control.trusty [hunk 3]
```diff
@@ -70,6 +70,7 @@ Depends: appscale-tools,
          python-pika,
          python-pillow,
          python-pip,
+         python-psutil,
```
Comment (cdonati) [inline debian/control.trusty]: Would you mind adding the wheezy package as well? It's named the same.

**Pull request merged**